### ISSUE TITLE
Failed to activate

### ISSUE THREAD

**Issue by Braunson** [twifty/atom-phpunit-integrator#36]
[Enter steps to reproduce:]

1. Install Plugin
2. Activate Plugin

**Atom**: 1.26.0 ia32
**Electron**: 1.7.11
**OS**: Microsoft Windows 10 Home
**Thrown From**: [php-unit-integrator](https://github.com/twifty/atom-phpunit-integrator) package 0.0.32

### Stack Trace

Failed to activate the php-unit-integrator package

```
At (0 , _util.promisify) is not a function

TypeError: (0 , _util.promisify) is not a function
    at /packages/php-unit-integrator/lib/util/php-unit-utils.js:9:19)
    at /packages/php-unit-integrator/lib/util/php-unit-utils.js:388:2)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.26.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.26.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.26.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/php-unit-integrator/lib/views/php-unit-integrator-view.js:14:35)
    at /packages/php-unit-integrator/lib/views/php-unit-integrator-view.js:27:43)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.26.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.26.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.26.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/php-unit-integrator/lib/php-unit-integrator.js:6:68)
    at /packages/php-unit-integrator/lib/php-unit-integrator.js:14:39)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.26.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.26.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.26.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/php-unit-integrator/lib/main.js:3:27)
    at /packages/php-unit-integrator/lib/main.js:7:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.26.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.26.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (~/AppData/Local/atom/app-1.26.0/resources/app/static/<embedded>:96:26)
    at Package.requireMainModule (~/AppData/Local/atom/app-1.26.0/resources/app/src/package.js:782:33)
    at Package.activateNow (~/AppData/Local/atom/app-1.26.0/resources/app/src/package.js:220:40)
    at measure (~/AppData/Local/atom/app-1.26.0/resources/app/src/package.js:206:33)
    at Package.measure (~/AppData/Local/atom/app-1.26.0/resources/app/src/package.js:88:25)
    at activationPromise.Promise (~/AppData/Local/atom/app-1.26.0/resources/app/src/package.js:200:20)
    at Promise (<anonymous>)
    at Package.activate (~/AppData/Local/atom/app-1.26.0/resources/app/src/package.js:198:38)
    at PackageManager.activatePackage (~/AppData/Local/atom/app-1.26.0/resources/app/src/package-manager.js:695:42)
    at ~/AppData/Local/atom/app-1.26.0/resources/app/node_modules/settings-view/lib/package-manager.js:491:35
    at exit (~/AppData/Local/atom/app-1.26.0/resources/app/node_modules/settings-view/lib/package-manager.js:128:22)
    at triggerExitCallback (~/AppData/Local/atom/app-1.26.0/resources/app/src/buffered-process.js:267:15)
    at ChildProcess.process.on (~/AppData/Local/atom/app-1.26.0/resources/app/src/buffered-process.js:297:15)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:194:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
```

### Commands

```
     -3:11.8.0 fuzzy-finder:toggle-file-finder (div.results-view.focusable-panel)
     -3:10 core:confirm (input.hidden-input)
     -3:08.1.0 intentions:highlight (input.hidden-input)
     -3:08.0 find-and-replace:show (input.hidden-input)
     -3:07.8.0 core:paste (input.hidden-input)
     -3:07.7.0 core:confirm (input.hidden-input)
     -3:01.4.0 intentions:highlight (input.hidden-input)
     -3:01.1.0 command-palette:toggle (input.hidden-input)
  2x -2:58.2.0 core:move-down (input.hidden-input)
     -2:54.7.0 core:confirm (input.hidden-input)
     -2:54.7.0 settings-view:install-packages-and-themes (input.hidden-input)
     -2:34.7.0 editor:consolidate-selections (input.hidden-input)
     -2:34.7.0 core:cancel (input.hidden-input)
     -1:16.6.0 core:confirm (input.hidden-input)
     -0:47.1.0 core:select-all (input.hidden-input)
  2x -0:46.1.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
aligner 1.2.4 
aligner-css 1.2.1 
aligner-javascript 1.3.0 
aligner-php 1.2.0 
aligner-scss 1.3.1 
atom-beautify 0.32.2 
atom-clock 0.1.16 
atom-ide-ui 0.11.0 
atom-shortcuts 0.0.2 
atom-typescript 12.3.8 
autocomplete-modules 2.0.0 
autocomplete-php 0.3.7 
autoprefixer 3.7.1 
blade-snippets 0.2.0 
busy-signal 1.4.3 
color-picker 2.3.0 
css-snippets 1.1.0 
docblockr 0.13.7 
editorconfig 2.2.2 
file-icons 2.1.18 
format-sql 0.3.0 
ftp-remote-edit 0.12.12 
highlight-bad-chars 1.0.7 
highlight-line 0.12.0 
highlight-selected 0.13.1 
html-id-class-snippets 1.4.1 
intentions 1.1.5 
language-blade 0.39.0 
language-sql-mysql 0.3.1 
language-vue 0.23.1 
language-vue-component 0.5.0 
larasnippets 0.44.0 
laravel 0.7.1 
linter 2.2.0 
linter-alex 4.0.0 
linter-jshint 3.1.6 
linter-php 1.5.1 
linter-phpcs 1.6.8 
linter-sass-lint 1.8.3 
linter-ui-default 1.7.1 
linter-write-good 0.9.0 
merge-conflicts 1.4.5 
minimap 4.29.8 
minimap-bookmarks 0.4.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-pigments 0.2.2 
minimap-selection 4.5.0 
php-cs-fixer 4.1.1 
php-debug 0.2.6 
php-unit-integrator 0.0.32 
pigments 0.40.2 
pinned-tabs 2.0.5 
print-atom 0.1.1 
project-manager 3.3.5 
qolor 0.4.2 
scroll-sync 0.2.4 
tabs-to-spaces 1.0.4 
teletype 0.13.0 
tree-view-git-modified 0.7.3 
vue-autocomplete 0.1.1 
vue-format 0.4.0 
vue2-autocomplete 0.1.0-alpha.4 
vuejs2-snippets 1.1.0 
```

**Comment by twifty** [comment]
which version of node are you running?
run: `node -v` from the cmd line

**Comment by twifty** [comment]
Fixed in 0.0.33
